www/include/views/graphs/graph-periods.html

```diff
@@ -11,7 +11,7 @@
     <div id="graphs" class="graph_content">
         <div class="graphZone">
           {foreach from=$periods item=period}
-          <div id="graph_wrapper_{$period.short}" data-graph-id="{$srv_id}" class="graph">
+          <div id="graph_wrapper_{$period.short}" data-graph-id="{$svc_id}" data-graph-period="{$period.short}" class="graph">
```

Comment:
what is the purpose of `data-graph-period` attribute ?

Reply:
@kduret it holds the period of the graph (day, week, month) since they are seperated.